src/ucp/rndv/proto_rndv.c

```diff
@@ -412,12 +412,18 @@ ucp_proto_rndv_bulk_init(const ucp_proto_multi_init_params_t *init_params,
 {
     ucs_status_t status;
     size_t mpriv_size;
+    size_t rndv_align_thresh =
```

Comment:
1. initialized vars come 1st
2. can have helper var for "context" instead. so this line will be shorter and line 424 still short enough